### PR TITLE
Add option to hide ProgressBar component from screen reader

### PR DESCRIPTION
**Pull request by aheitzmann**

### CODE REVIEW

File: components/progress_bar/ProgressBar.d.ts [hunk 1]
```diff
@@ -47,6 +47,16 @@ export interface ProgressBarProps extends ReactToolbox.Props {
    * @default false
    */
   disabled?: boolean;
+  /**
+   * Whether or not the progress bar should be hidden from the screen reader.
+   *
+   * Can be used if the caller wishes to provide more context for progress updates than just
+   * the percent. Also useful to work around an issue with IE11 + JAWS reading the aria-valuenow
+   * attribute instead fo teh aria-valuetext attribute on updates.
```
Comment (allinallin) [inline components/progress_bar/ProgressBar.d.ts]: `fo teh`

**Review by allinallin — COMMENTED** [review]
typooz

**Comment by allinallin** [comment]
squash the 3 commits in this PR with a push --force of just the single commit

File: components/progress_bar/ProgressBar.d.ts (hunk 1 above)
Comment (aheitzmann) [reply components/progress_bar/ProgressBar.d.ts]: done

**Review by allinallin — APPROVED** [review]

**Pull request merged**